recipes/bdwgc/all/patches/update-cmake-8_0_6.patch

```diff
@@ -323,11 +323,9 @@
 +  endif()
 +  include_directories(libatomic_ops/src)
 +  if (MSVC)
-+    # TODO: Rename the following to Atomic_ops
-+    find_package(libatomic_ops CONFIG)
-+    if (libatomic_ops_FOUND)
-+      # TODO: Rename the following to Atomic_ops::atomic_ops
-+      get_target_property(AO_INCLUDE_DIRS libatomic_ops::libatomic_ops
++    find_package(Atomic_ops CONFIG)
```

Comment:
Should this be `REQUIRED` to ensure the build fails if the package cannot be found? Then the `_FOUND` check wouldn't be necessary.

Reply:
In case of Conan - yes, but it general a client could install non-cmake-based variant of libatomic_ops (e.g. "apt get instal libatomic_ops-devel" on Ubuntu), in such case find_package(Atomic_ops CONFIG) won't find the cmake package but it just includes system-installed headers). The upstream master (not yet released) logic is here: https://github.com/ivmai/bdwgc/blob/master/CMakeLists.txt#L181

Reply:
Gotcha. It looks like you provide CMake config files in `libatomic_ops` now. Hopefully the package maintainers will pick those files up and include them in the `devel` packages in the future. Then it would be possible to simplify things by unifying around the `find_package()` behavior in CMake. Thanks for taking the time to explain that!